src/main/java/io/neonbee/endpoint/raw/RawEndpoint.java

```diff
@@ -178,6 +180,12 @@ queryParameterMap, multiMapToMap(request.headers()), routingContext.body().buffe
                         }
 
                         HttpServerResponse response = routingContext.response();
+
+                        Optional.ofNullable(context.responseData().get(RESPONSE_HEADERS))
```

Comment:
could `context.responseData()` be null?

Reply:
In this case, it's not possible because we're creating the context (DataContextImpl) object, and its implementation of responseData always returns a map.
But I could also write this null-save. I'll change that.